packages/playwright/.mocharc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
You can probably remove the values which are the default, i.e. everything except `timeout`.